tests/phpunit/tests/Config/TestStorage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace WPML\PB\Config;
+
+use OTGS\PHPUnit\Tools\TestCase;
+use WPML\WP\OptionManager;
+
+/**
+ * @group config
+ */
+class TestStorage extends TestCase {
+
+	const PB_KEY = 'the-pb-key';
+
+	/**
+	 * @test
+	 */
+	public function itShouldGet() {
+		$pbConfig = [ 'some PB config' ];
+
+		$optionManager = $this->getOptionManager();
```

Comment:
@strategio You can use the `OptionMock` trait for testing and then you don't need to mock OptionManager. You can just  use the Store get and set function to test.

```
$subject = $this->getSubject( new OptionManager() );
$subject->update( $pbConfig );

$this->assertEquals( $pbConfig, $subject->get() );
```

Reply:
@brucepearson, it's not so simple becaue [OptionManager](https://git.onthegosystems.com/wpml/sitepress-multilingual-cms/-/blob/develop/classes/options/class-wpml-option-manager.php) class is in Core.

Reply:
@strategio I forgot. I did have a plan to move this in WP library. I might do it sometime soon to make these things easier.